sdk/storage/storage-blob/README.md

```diff
@@ -216,12 +213,6 @@ async function main() {
     await streamToString(downloadBlockBlobResponse.readableStreamBody)
   );
 
-  // Delete container
-  await containerClient.delete();
-
-  console.log("deleted container");
-}
-
 // A helper method used to read a Node.js readable stream into string
 async function streamToString(readableStream) {
```

Comment:
Any plan to get rid of this help method? I remember in our first track 2 syncup, we agree to drop this.

Reply:
yeah, #2603 we were hoping that you could help implement it.

Reply:
Postponed.

---
In reply to: [293492865](https://github.com/Azure/azure-sdk-for-js/pull/3744#discussion_r293492865) [](ancestors = 293492865)